GEON-PPANG-iOS/Global/Network/Service/Search/SearchService.swift

```diff
@@ -6,3 +6,43 @@
 //
 
 import Foundation
+
+import Moya
+
+enum SearchService {
+    case searchBakery(bakeryID: String)
+}
+
+extension SearchService: TargetType {
+    var baseURL: URL {
+        return URL(string: Config.baseURL)!
+    }
+    
+    var path: String {
+        switch self {
+        case .searchBakery:
+            return URLConstant.search
+        }
+    }
+    
+    var method: Moya.Method {
+        switch self {
+        case .searchBakery:
+            return .get
+        }
+    }
+    
+    var task: Moya.Task {
+        switch self {
+        case .searchBakery(bakeryID: let bakeryID):
+            return .requestParameters(parameters: ["bakeryName" : bakeryID], encoding: URLEncoding.queryString)
```

Comment:
P2:
```suggestion
            return .requestParameters(parameters: ["bakeryName": bakeryID], encoding: URLEncoding.queryString)
```
린트에서도 워닝으로 감지하네여 ~~

Reply:
f1aa3973d36062ecf9599ec73b93c0be268f58c4